estate/models/estate_property_type.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models, fields
+
+class EstatePropertyType(models.Model):
```

Comment:
```suggestion
from odoo import models, fields


class EstatePropertyType(models.Model):
```
2 empty line before class definition